spec/helpers/sign_download_helper_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe SignsHelper, type: :helper do
+  drawing_1 = "1234/test01-1234-default.png"
+  drawing_2 = "1234/test02-1234.png"
+  drawing_3 = "1234/test03-default-1234-default.png"
+
+  describe '#convert_to_high_resolution' do
+    context 'when provided with a "default.png" sign drawing' do
+      it 'returns the sign drawing with the text "high_resolution" as part of the file name' do
+        expect(helper.convert_to_high_resolution(drawing_1)).to include("high_resolution.png")
+      end
+    end
+  end
+
+  describe '#stay_with_original_resolution' do
```

Comment:
just a heads up that the convention of `#name_of_a_thing` for a describe block is usually used to describe separate methods rather than scenarios, so you can probably get rid of this describe and the `'#convert_to_high_resolution and retain the string - default'` one below and just shift both of the context blocks into your `describe '#convert_to_high_resolution'` block above